modules/ztp-worker-node-applying-du-profile.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+// Epic CNF-5335 (4.11), Story TELCODOCS-643
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="ztp-additional-worker-apply-du-profile"]
+= Applying profiles to the worker node
+include::../_attributes/common-attributes.adoc[]
+//:context: additional-worker-sno
+
+You can configure the additional worker node with a DU profile.
+
+The expanded SNO with a DU profile is provisioned using the `PolicyGenTemplate` resources partitioned into common, group, and site-specific manifests. The Git repository for SNO that is linked to the ArgoCD `policies` application includes the following files:
```

Comment:
SNO cluster? SNO is already spelled out in the previous module, but DU is new here. Consider spelling out the first instance.